src/components/ChangePlan/ChangePlan.js

```diff
@@ -1,35 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useIntl } from 'react-intl';
 import { Helmet } from 'react-helmet';
-import { Card, CardPrice, CardAction, Ribbon } from './Card';
+import { Card, CardPrice, CardAction, Ribbon, CardFeatures } from './Card';
 import queryString from 'query-string';
 import { extractParameter } from '../../utils';
 
+function getPlanById(planId) {
+  return {
+    price: 15,
+    quantity: 1500,
+    name: 'Standard',
+    EmailParameterEnabled: true,
+    CancelCampaignEnabled: true,
+    SiteTrackingLicensed: false,
+    SmartCampaignsEnabled: false,
+    ShippingLimitEnabled: false,
+  };
+}
+
+function getPlanPlusByPlanId(planId) {
+  return {
+    price: 21,
+    quantity: 1500,
+    name: 'Plus',
+    id: 1,
+    EmailParameterEnabled: true,
+    CancelCampaignEnabled: true,
+    SiteTrackingLicensed: true,
+    SmartCampaignsEnabled: true,
+    ShippingLimitEnabled: true,
+  };
+}
+
+function getPlanUrl(planId, advancedPay, promoCode, _) {
+  return (
+    _('common.control_panel_section_url') +
+    `/AccountPreferences/UpgradeAccountStep2?IdUserTypePlan=${planId}&fromStep1=True&IdDiscountPlan=${advancedPay}&PromoCode=${promoCode}`
+  );
+}
+
 const ChangePlan = ({ location }) => {
   const promoCode = extractParameter(location, queryString.parse, 'promo-code') || '';
   const advancedPay = extractParameter(location, queryString.parse, 'advanced-pay') || 0;
   // TODO: remove planId, it just for testing right now
   const planId = parseInt(extractParameter(location, queryString.parse, 'planId')) || 0;
-  // Allow preselect by email or contacts
-  const selectedType = extractParameter(location, queryString.parse, 'selected-type') || '';
   const intl = useIntl();
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
-  const planUrl =
-    _('common.control_panel_section_url') +
-    `/AccountPreferences/UpgradeAccountStep2?IdUserTypePlan=${planId}&fromStep1=True&IdDiscountPlan=${advancedPay}&PromoCode=${promoCode}`;
-  const planPrice = 15;
-  const planQuantity = 100000;
-  const planName = 'Mensual';
 
-  const cardData = {
-    name: planName,
-    price: planPrice,
-    descriptionPlan: _('change_plan.until_x_subscribers', { subscribers: planQuantity }),
-    action: {
-      url: planUrl,
-      text: _('change_plan.calculate_price'),
-    },
+  const [isFeaturesVisible, setIsFeaturesVisible] = useState(false);
+  // Allow preselect by email or contacts
+  const selectedType = extractParameter(location, queryString.parse, 'selected-type') || '';
+  const currentPlan = getPlanById(planId);
+  const planPlus = getPlanPlusByPlanId(planId);
```

Comment:
Couldn´t you perhaps add both plans into an array so then you can map both plans into the same html? The one for the cards

Reply:
ready